keyboards/nobreakfast/nb62/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration(+1000)
```

Comment:
If you want to keep using `rules.mk`, please change the whole section to match https://github.com/qmk/qmk_firmware/blob/a239051c4a4779767059140892144dedea09aaf2/data/templates/avr/rules.mk (this is the last version of the `rules.mk` template before switching to the data-driven format as the default):
- fix the comment before the section, which is wrong in general (some options might be enabled by default and need an explicit `no` to disable);
- remove the obsolete `(+1000)`-like parts which refer to approximate code size (those values are often completely wrong);
- keep the order of options like in the template, add any extras below.